index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+   <meta charset="UTF-8">
+   <title>To do list</title>
+   <meta name="viewport" content="width=device-width,initial-scale=1">
+   <style></style>
+   <script></script>
+   <head>
+      <link rel="stylesheet" href="style.css">
+   </head>
+   <body>
+      <div class='container'>
+         <div class="menu">
+            <ul>
+               <li>
+                  <section id="Account">
+                     <table>
+                        <td><img src="pics/memoji.png" style='width: 120px; margin-left: -50px;'></td>
+                        <td>
+                           <p class="YourName">Hi Shobhit &#9995</p>
+                           <p class="AmountOfTasks">4 tasks pending</p>
+                        </td>
+                  </section>
+                  </table>
+               </li>
+               <li>
+                  <section id="TaskDo">
+                     <p>TaskDo</p>
+                  </section>
+               </li>
+               <li>
+                  <section id="Logout">
+                     <p>Logout</p>
+                  </section>
+               </li>
+            </ul>
+         </div>
+         <div class='content'>
+            <div class="NewTask">
+               <ul>
```

Comment:
it's not a list so that we can wrap it into a div tag
also, let's make this label an input field, and let's add "Add New Task" as an input placeholder